webpack.config.js

```diff
@@ -284,6 +284,9 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 					statsFilename: path.join( __dirname, 'stats.json' ),
 					statsOptions: {
 						source: false,
+						reasons: false,
```

Comment:
`reasons` should be set to `true`. Otherwise our `npm run whybundled` won't work. Seems like it was set to `false` again here https://github.com/Automattic/wp-calypso/commit/3660dfa95f9f47ff8b57160b8f7d282ca0f2bdbe#diff-11e9f7f953edc64ba14b0cc350ae7b9dL288

Reply:
Then maybe we could generate a different `stats.json` for `whybundled`? The `reasons` data are 280MB ☹️ That's too big an improvement to give it up easily 🙂

What if the `EMIT_STATS` variable supported multiple values? `false`, `true`, `"withreasons"`.

Reply:
> Seems like it was set to false again here

It wasn't set to `false` there -- the default value is `true`, so I just removed the redundant setting.

Reply:
Yes, let's make it optional so we can opt in with `whybundled`

Reply:
@flootr Updated in https://github.com/Automattic/wp-calypso/pull/27784/commits/881f5468a5f8299680235d9e2b2714ca1d15558f -- please have a look.

I tested `npm run whybundled` locally and it works 👌